rllib/models/base_model.py

```diff
@@ -0,0 +1,244 @@
+"""Model base classes.
```

Comment:
Please add RLlib annotations to the class. e.g. `@DeveloperAPI` `OverrideToImplementCustomization` etc.

Reply:
I don't really know what this tag means, wouldn't developers want to use every function in the class?

Reply:
Turns out `OverrideToImplementCustomLogic` actually errors if there are other decorators, like `@property`

Reply:
Lint will complain if you don't put it there :). I actually don't know the precise reason.